Rearranges the Science Department on IceBox Station for Toxins Gas Storage.

san7890 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This map change seeks to resolve issues with IceBox Station that have been present for a very, very, long time.

On the maps currently in rotation, Toxins Gas Storage is connected to the actual Toxins mixing laboratory. This is true for all of them, except IceBox Station. Toxins is extremely discouraging to do when you have to waste precious time running outside of your lab to get your cans of Oxygen and Plasma. It makes no sense for IceBox to be so radically different than Meta or Delta Station's Toxins, where there is usually a hallway facing door and a door that connects right to the mixing lab. Also, for some reason, IceBox has always just had a shitload of empty t1 canisters and portable scrubbers, which were extremely useless in the scope of Toxins, and probably any other aspect in the game.

This PR seeks to accomplish this change by swapping around the RD's office with Toxin's storage. This does shrink the RD's office a teeny bit, but everything was carried over. In order to reduce the amount of empty space in the Toxins Mixing Lab, I decided to expand Genetics' Monkey Pen, and actually add grass. (i don't know how genetics work please tell me if this is a powercreep so i can just reduce the monkey spawns or something) It also does reduce a bit of maintenance as well, but i think it's not as clunky as it previously was, and feels a bit smoother.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

As stated above, IceBox Science is so much more different than the other stations, and for no real good reason at all (than what I can think of). This PR reworks that part of Science in order to make it more in form with the other stations currently in rotation, and not have trained bomb-makers pull out their hair when people vote for IceBox.

This PR also doesn't add any additional gas canisters to Toxins (in comparison to what was already there), it just shifts it to another place.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Bombmakers unite! Nanotrasen has recently retrofitted the Science department on the IceBox Station and made the Toxins Gas Storage room adjacent to... Toxins! They had to sell all of those scrubbers and gas canisters to pay for it though.
del: The RD however, is not too jazzed about his new office on IceBox, It's cut down by an incredibly slim margin, but...... What's the upside again?
add: Genetics Monkey Pen has shifted... Odd...
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Before:
![image](https://user-images.githubusercontent.com/34697715/92334241-62dab280-f049-11ea-9bea-100cea36e423.png)

After:
![image](https://user-images.githubusercontent.com/34697715/92334245-6a01c080-f049-11ea-98fa-ed0366443011.png)

Area Overlay:
![image](https://user-images.githubusercontent.com/34697715/92334248-70903800-f049-11ea-9925-154d3dd0ed0b.png)

(this is my first PR go easy on me)

san7890 committed:
Update IceBoxStation.dmm

yes

san7890 committed:
fixes the missing holopad in the RD office

screenshots are now outdated hm yes

san7890 committed:
removes random step_x variable

travis say i do stinky poopoo?

(uses Notepad++ like a boss)

why yes i am something of a coder myself

wesoda25 commented:
I kinda always liked that little room off to the side with all the canisters, it was something seemingly unique. Guess that’s not really an argument though so 👍

tralezab commented:
Naw wesoda that's a bit valid, there is something so fucking fun about turning the canister closet into the hell box

san7890 commented:
> Naw wesoda that's a bit valid, there is something so fucking fun about turning the canister closet into the hell box

you can still make a wacky ass hellbox in those canister room, it just has two doors now

Hamcha commented:
I love the strategic placement of the O2/Plasma canisters!

My only gripe: I kinda feel bad that a lot of space translated into that huge monkey pen, does gene even need that many monkeys at roundstart?

san7890 commented:
> I love the strategic placement of the O2/Plasma canisters!
> 
> 
> 
> My only gripe: I kinda feel bad that a lot of space translated into that huge monkey pen, does gene even need that many monkeys at roundstart?

I tried to expand maints, but it all just looked bad. I ended up committing to the namesake of Ice Box, and just boxed out all the rooms. I think Meta has the same amount of roundstart monkeys, correct me if I'm wrong on that.

san7890 committed:
fixed problems with modular consoles and RD consoles

i don't know how this issue keeps popping up but i hope it doesn't happen again

shout outs to map diff

san7890 committed:
forgot power cable

when you forget to ctrl-click add an object

(hold up)

san7890 commented:
AppVeyor appears to have had a stroke, it compiles fine as of the last commit.

Tlaltecuhtli commented:
but a fire extinguisher cabinet inside the toxin storage

san7890 commented:
> but a fire extinguisher cabinet inside the toxin storage

The original Toxins storage didn't have one in the room, but rather right outside the door. I replicated this by having either entrance to the room have a fire extinguisher.

san7890 committed:
adds fire extinguisher, toxins storage APC, and cleans up obj's

i don't know how i forgot the APC

bruh

san7890 committed:
fixed tiles

maybe THIS one will fix the RD console turfs

san7890 committed:
OK THIS TIME

THIS TIME I'LL MAK E THOS GREEN DECALS WORK AAAA

san7890 committed:
changed cameras to autoname variants

forgot (FIND OUT HOW CAMERAS WORK) was on my list of "how 2 map lol"

san7890 committed:
why oh why must i forget to save

that moment when you refuck a change you have already unfucked because you got too fucking lazy to save

i will yank my hair out if i have to commit this same exact green tile again bro

Fikou commented:
the monky pen is ugly
the rd office is really weird now, just random computers, no chair or anything, and its missing a wall
the canisters are kinda weirdly placed?

san7890 commented:
> the monky pen is ugly

How would you recommend I fix that?
> the rd office is really weird now, just random computers, no chair or anything, and its missing a wall

I forgot to add the chair, my bad. What wall is it missing? I'm not at my computer so I'm just looking at the renders. I had a different setup for the computers that fit a lot better, but the whole thing involving the RD's facehugger just fucks with the whole flow. I feel like a cramped, stuffy office, encapsulates the essence of the RD though. A geeky nerd in a beige suit wants an office cramped full of all his geeky shit, with computers and miscellaneous items strewn about.
> the canisters are kinda weirdly placed?

I might remove a singular can of plasma to make the co2 canister fit in right. Four is plenty.

I'm away from my computer right now, but I'll look over and incorporate stuff I've fluked tommorow.

Fikou commented:
just keep the monkey pen the same as it was before and enlargen the room by 2 and fill it with some more canisters and scrubbers

san7890 commented:
Sounds good but:
> fill it with some more canisters and scrubbers
![image](https://user-images.githubusercontent.com/34697715/92315645-e93db880-efa5-11ea-8dbb-b8bd41ff38ae.gif)

I've got an idea on how to handle it though.
> the wall in rd's is missing in the window part

I have no fucking idea what causes this. The map render on the latest commit says it's fine (not missing). Is there an /obj/structure/blob/core on the tile where the computer is? For some reason, that kept coming back up and removing a window. I thought I had fixed it, and I hope it's not a code issue.

san7890 committed:
fixed stuff

epic gamer style

-rd now has chair
-monkey pen is a lot quirkier now
-moved air pumps back
-tell me if anything is fucked

san7890 commented:
alright, i figured it all out today and pushed the changes here. the monkey pen isn't "weird" anymore, and I've corrected any other flaws. please let me know if you need any changes done.

san7890 commented:
@ShizCalev is there anything else that needs to be done?

Timberpoes reviewed (approved):
Yo! Sorry for the radio silence. You've not been forgotten, or if you were you now are not.

Tentatively approving these changes. 12-24 hour delay before I merge incase any pressing issues crop up. If any maintainer wants to block this, I'm happy to cede.

The reason I'm only tentatively approving is because our stations are different and are meant to be different. If everything had the same layout, then why do we have different stations at all?

However, change is sometimes good. Changes like this mix things up, break familiarity and can make people feel a bit uneasy, which is a pretty cool feeling to have in a paranoia game like SS13.

So I'm going this a classic Timberpoes Thumbs Neither Up Nor Down and will look to merge this as per my above timescale.

san7890 commented:
> Yo! Sorry for the radio silence. You've not been forgotten, or if you were you now are not.
> 
> Tentatively approving these changes. 12-24 hour delay before I merge incase any pressing issues crop up. If any maintainer wants to block this, I'm happy to cede.
> 
> The reason I'm only tentatively approving is because our stations are different and are meant to be different. If everything had the same layout, then why do we have different stations at all?
> 
> However, change is sometimes good. Changes like this mix things up, break familiarity and can make people feel a bit uneasy, which is a pretty cool feeling to have in a paranoia game like SS13.
> 
> So I'm going this a classic Timberpoes Thumbs Neither Up Nor Down and will look to merge this as per my above timescale.

thanks for the reply. I get your point that diversity in maps makes them each uniquely different, but Box Science is already much more different than that of Meta or Delta. This change only addresses the fact that one of the key components to a specialty of Science is unnecessarily further away from the actual workplace, and is not conducive at all. It's like if the Genetics monkey pen was a 15 second walk away, and in another section of science than where the DNA consoles are. It discourages the level of innovation that Toxins can achieve already (50k points, anomaly cores) by adding an unnecessary complexity. 

I'll happily take the late-merge over any sort of "pocket close", though.

Timberpoes commented:
Looks like I got pre-empted! Looks like Jared liked it enough to merge it. Your lucky day!
